rfcs/text/0000-alternative-planning.md

```diff
@@ -0,0 +1,315 @@
+
+In addition, theme teams have greater flexibility to adapt to changing user circumstances and implementation discoveries
+than per-component teams, especially if they are working to an indicator rather than detailed roadmap goals.
+
+Pros:
+1) Theme teams can consider implementations that optimize across the whole of the DCP, rather than being incentivized to
+optimize their local component at the expense of cross-DCP optimization.
+2) Theme teams are more flexible to adapt their work as circumstances emerge and change, rather than requiring a 
+planning cycle.
+3) It may be easier for developers to move between theme teams as required, rather than being tied to components that
+face varying workloads.
+4) It will be easier to create theme teams to work on cross-cutting DCP objectives, such as metrics of reliability or
+whole-DCP dashboards.
+5) More developers get a chance to lead if there are theme teams.
+6) Theme teams grow their familiarity with the DCP as a whole. There are no longer per-component silos, though one 
+would have to avoid per-theme silos emerging.
+
+Cons:
+1) Communication has to become more asynchronous and/or the overlap between UK and US West Coast day has to be increased
+as much as possible bearing in mind cultural variation and work-life balance. Otherwise, more theme teams may
+mean even more pressure on limited communication bandwidth.
+2) Theme teams may be smaller than per-component teams - developers moving teams may be a political challenge.
+3) Theme teams may be more isolated. For example, if 3 members are in the US and only 1 in the UK.
+4) It's a risky change that requires commitment and adaptation across the entire DCP. To reduce the risk it could be
+done first on an experimental basis with a single team, or even done partially with smaller 'maintainer' teams still
+nurturing the individual components.
+
+#### Best practice is evangelized rather than mandated
+This approach recognizes that there is *cultural variation* between teams. It assumes that imposing a process will not bring
+alignment because there is *no overall control*. Teams also face different circumstances which means one size will not
+fit all. Alignment is valuable but comes about through evangelization, discussion of shared problems and spontaneous convergence.
+
+Pros:
+1) Teams can adapt their workflow to fit their circumstances, both at the present time and as those circumstances 
+change over time.
+2) Alignment is strong because it comes about through choice and agreement.
+
+Cons:
+1) It's more work for project managers since the scope for tracking status automatically is reduced as team
+workflows are less aligned.
+ 
+#### A culture of individual empowerment
+People are afraid to take risks in this project, despite our encouragement. I've seen this many times in my relatively
+short time with the DCP. Why is this? One reason is that people find the de facto mechanism of cross-component change,
```

Comment:
Can you offer some concrete examples where people were afraid to take risks?

Reply:
No. If people didn't want to take risks because they didn't want to deal with the process then I'm not going to name them either unless they want to be named. Either people agree from their own experience that my statement is true or they don't.